tests/integration/test_name.py

```diff
@@ -0,0 +1,55 @@
+named_app = """ 
+library(dash)
+library(dashHtmlComponents)
+app <- Dash$new()
+
+app$title("Testing")
+
+app$layout(htmlDiv(list(htmlDiv(id='container',children='Hello Dash for R testing'))))
+app$run_server()
+"""
+
+app_with_template = """
+library(dash)
+library(dashHtmlComponents)
+app <- Dash$new()
+
+string <-
+  "<!DOCTYPE html>
+        <html>
+          <head>
+            {%meta_tags%}
+            <title>Testing Again</title>
```

Comment:
What happens if you set both `app$title` and the template? The template wins?

Reply:
If you set `app$title`, it wins. I suspect this is what we want, rather than the template taking precedence, though I cannot say for sure.